.github/workflows/s3-bucket.yml

```diff
@@ -0,0 +1,44 @@
+name: S3 Bucket Test
+
+on:
+  push:
+  workflow_dispatch:
```

Comment:
Don't we want to run this on pull requests?
https://github.com/mapbox/node-pre-gyp/blob/a2785e88e6fec291b0df015da078deb0d00ba1ba/.github/workflows/ci.yml#L5-L8